src/components/Trade/hooks/useSwapper/useSwapper.ts

```diff
@@ -191,7 +195,7 @@ export const useSwapper = () => {
       throw new Error(`unsupported chain id ${sellAsset.chainId}`)
     })()
 
-    setFees(result, sellAsset)
+    await setFees(result, sellAsset)
```

Comment:
Can we rename this one to `setFormFees` to make it clear that this is a wrapper around react-hook-form that will do some arbitrary logic before calling `setValue`, as opposed to a `setValue` call, which by itself is synchronous?
This is probably why the `await` was forgotten in the first place, and with it added, I had to double check the signature to figure out why this method is async, but the following `setValue` isn't

Reply:
I like it. Addressed in https://github.com/shapeshift/web/pull/2107/commits/1396be4d1b69881217749260a0754b006b3fec38.